lib/teaspoon/engine.rb

```diff
@@ -98,7 +98,7 @@ def render_exception(_env, exception)
 
 begin
   require 'action_view'
-  if ActionView::VERSION::STRING == '4.2.5'
+  if ActionView::VERSION::STRING.start_with?('4.2.5')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.